src/implementations/vanilla/components/mega-menu/mega-menu.js

```diff
@@ -982,6 +987,8 @@ export class MegaMenu {
         'ecl-mega-menu__subitem--expanded',
         'ecl-mega-menu__subitem--current',
       );
+      const itemLink = queryOne(this.subLinkSelector, level2);
```

Comment:
confusing, we were and we are still setting the aria-expanded on the list items as well, if we set it on the inner link and if that makes sense then i guess we should remove it from the list item

Reply:
that's true, but lots of code is depending on the aria-expanded at list item level (js and css).
I didn't want to do major changes now as the mega menu will have to be updated soon anyways